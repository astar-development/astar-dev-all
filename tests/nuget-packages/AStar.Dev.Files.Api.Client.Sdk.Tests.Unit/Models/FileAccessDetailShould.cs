using AStar.Dev.Files.Api.Client.SDK.Models;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

public sealed class FileAccessDetailShould
{
    [Fact]
    public void ReturnTheExpectedToString() =>
        new FileAccessDetail().ToString()
                              .ShouldMatchApproved();
}
