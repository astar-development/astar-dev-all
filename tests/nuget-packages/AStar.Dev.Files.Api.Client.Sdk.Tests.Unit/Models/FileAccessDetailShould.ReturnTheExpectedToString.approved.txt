{
  "id": 0,
  "detailsLastUpdated": null,
  "lastViewed": null,
  "softDeleted": false,
  "softDeletePending": false,
  "moveRequired": false,
  "hardDeletePending": false
}